tensorflow/java/src/main/java/org/tensorflow/Session.java

```diff
@@ -225,7 +225,7 @@ public Runner addTarget(String operation) {
       public Reference() {
         synchronized (nativeHandleLock) {
           if (nativeHandle == 0) {
-            throw new IllegalStateException("run() called after the Session was close()d");
+            throw new IllegalStateException("run() called after the Session was closed");
```

Comment:
Would it be clearer to say:
"run() cannot be called on the Session after close()"
?